__tests__/integration/__snapshots__/one-app.spec.js.snap

```diff
@@ -42,6 +42,44 @@ Object {
 }
 `;
 
+exports[`Tests that require Docker setup one-app successfully started metrics has all metrics 1`] = `
```

Comment:
I'd encourage looking at the diff of this file across the two commits to see the difference in metric names